spec/system/publishers_can_edit_a_published_vacancy_as_a_school_spec.rb

```diff
@@ -111,7 +111,8 @@
         fill_in "publishers_job_listing_pay_package_form[salary]", with: ""
         click_on I18n.t("buttons.update_job")
 
-        within_row_for(text: I18n.t("helpers.label.publishers_job_listing_pay_package_form.salary")) do
+        within_row_for(text: I18n.t("helpers.label.publishers_job_listing_pay_package_form.salary"),
+                       element: ".govuk-checkboxes__label") do
           expect(page).to have_content(I18n.t("pay_package_errors.salary.blank"))
```

Comment:
I wonder, is this checking within this row, or within the page as a whole if you use `page`?

I wonder if you could use a block parameter instead?

Something like

```
within_row_for(text: I18n.t("helpers.label.publishers_job_listing_pay_package_form.salary"), element: ".govuk-checkboxes__label") do |row|
          expect(row).to have_content(I18n.t("pay_package_errors.salary.blank"))
```

Reply:
Ah yeah I think this is to ensure we get the error message on the field as well as in the summary up top:
<img width="880" alt="image" src="https://user-images.githubusercontent.com/24639777/184924254-3a880766-663e-40ad-8d4e-9697448d9649.png">